packages/interfaces/src/relay.ts

```diff
@@ -9,20 +8,17 @@ import type {
 } from "./message.js";
 import type { Callback, SendResult } from "./protocols.js";
 
-export interface RelayEvents {
-  "observer:added": CustomEvent;
-  "observer:removed": CustomEvent;
-}
+type PubSubTopic = string;
+type ContentTopic = string;
```

Comment:
future work: Would be good to actually use these two types across the codebase.

Reply:
Good point from the perspective of improving readability but I would try to refrain from pushing it's usage on the consumers since these types are primitive for now. 

Here is the issue #1254